src/app/ReadClient.cpp

```diff
@@ -1174,5 +1175,18 @@ CHIP_ERROR ReadClient::GetMinEventNumber(const ReadPrepareParams & aReadPrepareP
     }
     return CHIP_NO_ERROR;
 }
+
+void ReadClient::TriggerResubscribeIfScheduled(const char * reason)
+{
+    if (!mIsResubscriptionScheduled)
```

Comment:
What sets this?

Reply:
`ReadClient::ScheduleResubscription` right when it sets the resubscription timer.